src/Web/src/components/groups/api/getGroups/GetGroupsRequest.ts

```diff
@@ -12,7 +12,7 @@ export class GetGroupsRequest extends RequestCore {
 		super({
 			properties: {
 				method: RequestType.GET,
-				endpoint: init.userOnly ? RequestEndpoint.GET_USER_GROUPS : RequestEndpoint.GET_ALL_GROUPS,
+				endpoint: init.userOnly ? RequestEndpoint.GET_ALL_GROUPS : RequestEndpoint.GET_ALL_GROUPS,
```

Comment:
to była zamierzona zmiana? teraz niezależnie od warunku userOnly idzie ten sam endpoint